scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -1394,149 +1415,148 @@ def _init_props(self, default_prop_types: bool = False,
                 if name not in _DEFAULT_PROP_SPECS:
                     continue
                 prop_spec = _DEFAULT_PROP_SPECS[name]
-                val = self._prop_val(name, prop_spec.type, False, False, None,
-                                     None, err_on_deprecated)
-                self.props[name] = Property(prop_spec, val, self)
+                prop = self._node.props.get(name)
+                val = self._prop_val(prop_spec, prop, err_on_deprecated)
+                self.props[name] = Property(name, prop_spec, val, self)
 
     def _init_prop(self, prop_spec: PropertySpec,
                    err_on_deprecated: bool) -> None:
         # _init_props() helper for initializing a single property.
```

Comment:
Stale comment. This function now initializes more than one property.
It could also use a new name, like maybe `_init_props_from_spec()`.